easybuild/tools/options.py

```diff
@@ -1471,14 +1471,25 @@ def parse_external_modules_metadata(cfgs):
     """
     Parse metadata for external modules.
 
-    :param cfgs: list of config files providing metadata for external modules
+    :param cfgs: list of (glob patterns for) paths to config files providing metadata for external modules
```

Comment:
https://github.com/easybuilders/easybuild-framework/blob/4ac2f6d2add3aed853430f97e87415a7e7b717af/easybuild/tools/options.py#L452 should also be updated?